src/api/routes/articleRouter.js

```diff
@@ -154,4 +155,12 @@ articleRouter.delete('/:slug/comments/:id',
   checkCommentOwner,
   commentController.deleteComment);
 
+articleRouter.post('/:slug/:id/report/comment',
```

Comment:
@MCFrank16 a better url is: `/:slug/comments/:comment_id/report`

Reply:
yeah this one looks better, lemme change it real quick

Reply:
but @MemunaHaruna the ``:comment_id`` in the route won't work unless I rename it to ``: id`` because I am using middleware to check if the comment I am trying to report exists, and that middleware and even the comment controllers uses 🆔 in the route path. so if I change it, I will have to change everything in the comments controller. @nkpremices please confirm this.

Reply:
`:id` is fine @MCFrank16